### PR TITLE
Fixing links

### PR DESCRIPTION
**Pull request by lauramthomas**
**Thank you for contributing to the `sauce-docs` project!**
**A well described pull request helps maintainers quickly review and merge your change**

Before submitting your PR, please check our [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD)
guidelines, applied for this repository. Avoid large PRs, help reviewers by making them as simple 
and short as possible.

### Description
Changed several links from wiki to docs.saucelabs.com

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/basics/sso/setting-up-single-sign-on.md [hunk 1]
```diff
@@ -34,10 +34,10 @@ We recommend setting it to the email address, as this makes it easier to manage
 If the **saml:NameID** field contains only a user name and no @ symbol, then the SSO username will be based on your Domain Name. If your domain was sauce and the field value was **john.smith**, the SSO user name would be **sso-sauce-john.smith**.
 
 ## AudienceRestriction
-This is a required attribute within the SAML assertion that indicates the specific users the assertion is intended for. It must be equal to the value of entityID from SauceLabs metadata based on your data center. For details, see the **Single Sign-On Configuration** section in [Data Center Endpoints](https://wiki.saucelabs.com/display/DOCS/Data+Center+Endpoints).
+This is a required attribute within the SAML assertion that indicates the specific users the assertion is intended for. It must be equal to the value of entityID from SauceLabs metadata based on your data center. For details, see the **Single Sign-On Configuration** section in [Data Center Endpoints](/dev/cli/sauce-connect-proxy/index.html#data-center-endpoint).
```
Comment (sweeneyskirt-sl) [inline docs/basics/sso/setting-up-single-sign-on.md:37]:
```suggestion
This is a required attribute within the SAML assertion that indicates the specific users the assertion is intended for. It must be equal to the value of entityID from SauceLabs metadata based on your data center. For details, see the **Single Sign-On Configuration** section in [Data Center Endpoints](/dev/cli/sauce-connect-proxy#data-center-endpoint).
```

File: docs/basics/sso/setting-up-single-sign-on.md [hunk 2]
```diff
@@ -34,10 +34,10 @@ We recommend setting it to the email address, as this makes it easier to manage
...
 
 ## Configuration Information for SSO
-The Signing/Encryption Certificate and Entity AssertionConsumeURLs required for configuring Sauce Labs SSO vary based on your data center. For details, see [Data Center Endpoints](https://wiki.saucelabs.com/display/DOCS/Data+Center+Endpoints). Here are some things to be aware of:
+The Signing/Encryption Certificate and Entity AssertionConsumeURLs required for configuring Sauce Labs SSO vary based on your data center. For details, see [Data Center Endpoints](/dev/cli/sauce-connect-proxy/index.html#data-center-endpoint). Here are some things to be aware of:
```
Comment (sweeneyskirt-sl) [inline docs/basics/sso/setting-up-single-sign-on.md:40]:
```suggestion
The Signing/Encryption Certificate and Entity AssertionConsumeURLs required for configuring Sauce Labs SSO vary based on your data center. For details, see [Data Center Endpoints](/dev/cli/sauce-connect-proxy#data-center-endpoint). Here are some things to be aware of:
```

File: docs/mobile-apps/live-testing/live-mobile-app-testing.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ With Sauce Labs, you can test your mobile applications on a variety of real Andr
 * A mobile app to test. If you don't have one, you can use the [Sauce Labs sample mobile app](https://github.com/saucelabs/sample-app-mobile).
 
 ### Uploading an App
-You can upload your app via the Sauce Labs UI or via the REST API. For information about uploading via the API, see [Upload Files with the REST API](https://wiki.saucelabs.com/display/DOCS/Application+Storage#ApplicationStorage-UploadFileswiththeRESTAPI).
+You can upload your app via the Sauce Labs UI or via the REST API. For information about uploading via the API, see [Upload Files with the REST API](/mobile-apps/app-storage/index.html#uploading-apps-via-rest-api).
```
Comment (sweeneyskirt-sl) [inline docs/mobile-apps/live-testing/live-mobile-app-testing.md:18]:
```suggestion
You can upload your app via the Sauce Labs UI or via the REST API. For information about uploading via the API, see [Upload Files with the REST API](/mobile-apps/app-storage#uploading-apps-via-rest-api).
```

**Review by sweeneyskirt-sl — COMMENTED** [review]
Am I the one who put these bad links in here with my previous review?!?! <shame!/>

File: docs/secure-connections/ipsec-vpn.md [hunk 4]
```diff
@@ -134,7 +134,7 @@ To ensure compliance with your company's settings and network policy, we recomme
 
 #### __Public Real Device Testing__
 
-To run tests on public real devices in the Sauce Labs cloud using Sauce IPSec Proxy, your organization admin must switch on **Enable Sauce Connect Proxy/IPSec VPN for Public Cloud Devices** (see [Security Settings](/basics/acct-team-mgmt/org-settings.md)), a security setting that is disabled by default.
+To run tests on public real devices in the Sauce Labs cloud using Sauce IPSec Proxy, your organization admin must switch on **Enable Sauce Connect Proxy/IPSec VPN for Public Cloud Devices** (see [Security Settings](/basics/acct-team-mgmt/org-settings)), a security setting that is disabled by default.
```
Comment (walkerlj0) [inline docs/secure-connections/ipsec-vpn.md:137]:
```suggestion
To run tests on public real devices in the Sauce Labs cloud using Sauce IPSec Proxy, your organization admin must switch on **Enable Sauce Connect Proxy/IPSec VPN for Public Cloud Devices** (see [Security Settings](/basics/acct-team-mgmt/org-settings)), which is a security setting that is disabled by default.
```

**Pull request merged**